aptos-move/framework/src/built_package.rs

```diff
@@ -265,6 +266,12 @@
             }
         }
 
+        if let Some(model_options) = model.get_extension::<Options>() {
+            if model_options.experiment_on(Experiment::STOP_AFTER_EXTENDED_CHECKS) {
```

Comment:
Experiments are not intended to control upstream tools.

Reply:
This is not really an "upstream tool" since it's where extended_checks is run.